test/unittests/NDB_Factory_Test.php

```diff
@@ -236,6 +239,12 @@ function testProject()
      */
     function testCandidate()
     {
+        $mockdb = $this->getMockBuilder("\Database")->getMock();
```

Comment:
These tests were failing in the same way as #7199 so I stole the fix from there. I'm not sure why they're failing on this branch it seems to be unrelated, but was necessary for the tests..